tox.ini

```diff
@@ -24,7 +24,7 @@ deps =
     pytest
     pytest-cov
 commands =
-    pytest
+    pytest --cov=./
```

Comment:
```suggestion
    pytest 
```

The pytest configuration is already done in setup.cfg:

https://github.com/RIOT-OS/riotctrl/blob/4b57c84bdbc12b1b614d2343602e133704202edd/setup.cfg#L1-L6

Reply:
Ahhh that is where the magic happens. I was not sure where you did it in riot-generator.